rules/integrations/kubernetes/privilege_escalation_container_created_with_excessive_linux_capabilities.toml

```diff
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2022/09/20"
+integration = "kubernetes"
+maturity = "production"
+min_stack_comments = "New fields added to Kubernetes Integration"
+min_stack_version = "8.4.0"
+updated_date = "2022/09/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects a container deployed with one or more dangerously permissive Linux capabilities that can be used for
```

Comment:
Great thorough description. How do you feel about moving some of it to the investigation guide and expanding on it

Reply:
That's a good idea, will do!